source/ingress_test.go

```diff
@@ -191,8 +192,9 @@ func testEndpointsFromIngress(t *testing.T) {
 			},
 			expected: []*endpoint.Endpoint{
 				{
-					DNSName: "foo.bar",
-					Targets: endpoint.Targets{"8.8.8.8"},
+					DNSName:    "foo.bar",
+					RecordType: endpoint.RecordTypeA,
```

Comment:
what about adding some tests for `endpoint.RecordTypeAAAA`? I feel like we only slightly adjust the old tests being passed but not adding new tests for testing AAAA

Reply:
I added a test case for extracting an AAAA from an Ingress status. Are there any other areas where you think this is lacking coverage?

Reply:
Looks like it is tested in https://github.com/kubernetes-sigs/external-dns/pull/2461/files#diff-7ec1f21be9321a28b8532f54f2da218bb0d7541c9af60caa3e6b6dd10b89d73cR426 , so lgtm